core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -237,16 +238,12 @@ protected void addPublicKeysListHandler(
 
   protected void addReloadHandler(
       final Router router,
-      final ArtifactSignerProvider artifactSignerProvider,
+      final List<ArtifactSignerProvider> artifactSignerProvider,
```

Comment:
nit: As per other comment, think this could be explicitly ArrayList to express the ordered property and name should change to reflect it

```suggestion
      final ArrayList<ArtifactSignerProvider> orderedArtifactSignerProviders,
```

Reply:
This should stay as a type of `List` instead of changing to an `ArrayList`. As the `List` type is by definition ordered. Also rather not tie this to a specific implementation of List as it doesn't matter for the reloadHandler

Reply:
I was wrong, List is indeed ordered, I must have been thinking of Collection :)